charts/orchestrator/values-rc.yaml

```diff
@@ -0,0 +1,22 @@
+isReleaseCandidate: true # Indicates RC builds should be used by the chart to install RHDH and Sonataflow
+ocpVersion: v4.15 # The version of the target OCP cluster. Supported versions are: v4.13, v4.14, v4.15, v4.16
```

Comment:
why can't we fetch the version on the fly instead using the clusterversion CR?
```
$> oc get clusterversion
NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.14.23   True        False         18d     Cluster version is 4.14.23
```
to get the version field, the CRD uses this jsonpath:
`{.status.history[?(@.state=="Completed")].version}`

Or even more simple is to fetch the channel value from the same object:
```
oc get clusterversion version -oyaml
apiVersion: config.openshift.io/v1
kind: ClusterVersion
metadata:
  name: version
...
spec:
  channel: stable-4.14
  clusterID: 2c22c433-0a00-4342-91a8-131ee2fdcf10
...
```